app/controllers/system_admin/dttp_trainees_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module SystemAdmin
+  class DttpTraineesController < ApplicationController
+    def show
+      trainee = Trainee.from_param(params[:id])
+      render json: trainee.dttp_trainee.response
```

Comment:
```suggestion
      render json: trainee.dttp_trainee&.response
```

and the same for the placement assignments i guess?